### PR TITLE
Created api to return batch of files by paths

### PR DESCRIPTION
**Pull request by monassfaro**
# What? Why?
Create API endpoint to return batch of configuration files by paths.

### CODE REVIEW

File: src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigurationClientResource.java [hunk 1]
```diff
@@ -139,6 +143,17 @@ public ResponseEntity<Void> refreshConfiguration(HttpServletRequest request) {
         return ResponseEntity.ok().build();
     }
 
+    @PostMapping(value = PROFILE + "/configs_map")
+    @Timed
+    @LoggingAspectConfig(resultDetails = false)
+    public ResponseEntity<Map<String, Configuration>> getConfigurationsByPaths(@RequestBody List<String> paths) {
+        List<String> nonNullPaths = Optional.ofNullable(paths)
+            .orElseGet(Collections::emptyList);
```
Comment (sergeysenja1992) [inline src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigurationClientResource.java:151]: better to one line имхо

File: src/test/java/com/icthh/xm/ms/configuration/service/ConfigurationServiceUnitTest.java [hunk 2]
```diff
@@ -109,4 +116,32 @@ public void refreshTenantConfigurations() {
...
+        when(configProxyRepository.find(firstPath, null)).thenReturn(new ConfigurationItem("firstCommit", firstConfig));
+        when(configProxyRepository.find(secondPath, null)).thenReturn(new ConfigurationItem("secondCommit", secondConfig));
+
+        Map<String, Configuration> actual = configurationService.findConfigurations(List.of(firstPath, secondPath));
+
+        assertThat(actual).hasSize(2);
+        assertThat(actual.get(firstPath)).isEqualTo(firstConfig);
+        assertThat(actual.get(secondPath)).isEqualTo(secondConfig);
+
+        verify(configProxyRepository, times(2)).find(any(), any());
```
Comment (sergeysenja1992) [inline src/test/java/com/icthh/xm/ms/configuration/service/ConfigurationServiceUnitTest.java:135]: don't use any()

File: src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigurationClientResource.java (hunk 1 above)
Comment (monassfaro) [reply src/main/java/com/icthh/xm/ms/configuration/web/rest/ConfigurationClientResource.java]: Fixed

File: src/test/java/com/icthh/xm/ms/configuration/service/ConfigurationServiceUnitTest.java (hunk 2 above)
Comment (monassfaro) [reply src/test/java/com/icthh/xm/ms/configuration/service/ConfigurationServiceUnitTest.java]: Replaced by eq() and isNull() methods

**Review by sergeysenja1992 — APPROVED** [review]

**Review by major13ua — APPROVED** [review]

**Pull request merged**